MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,899 @@
+
+You can change the order of item's within a row using `ItemsOrderInRow` property, which is applicable only when `ColumnCount` is set to more than one. You can place an item within a row from 0 to `ColumnCount` - 1 position.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ColumnCount = 3;
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 1;
+                e.DataFormItem.ItemsOrderInRow = 2;  
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Add Label icon for the editor
+
+You can set label icon instead of using label text by using GenereateDataFormItem event when `AutoGenerate` is true.
+
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "PhoneNumer")
+            {
+                e.DataFormItem.LeadingLabelIcon = new FontImageSource() 
+                { 
+                    Glyph = "K", 
+                    Color = Colors.Black, 
+                    FontFamily = "DataFormIcons", 
+                    Size=18 
+                };
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Change the Text style 
+
+You can change the text style for label text, editor text , valid message text and error message text by using GenereateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add refrence link for explict create data editor
+<table>
+<tr>
+<th>Text</th>
+<th>Property</th>
+</tr>
+<tr>
+<td>Label Text</td>
+<td>LabelTextStyle</td>
+</tr>
+<tr>
+<td>Editor Text</td>
+<td>EditorTextStyle</td>
+</tr>
+<tr>
+<td>Error Message Text</td>
+<td>ErrorLabelTextStyle</td>
+</tr>
+<tr>
+<td>Valid Message Text</td>
+<td>ValidMessageLabelTextStyle</td>
+</tr>
+</table>
+
+
+by using DataFormTextStyle class properties, you can change the text style .
+
+DataFormTextStyle class properties:-
+    1.FontSize
+    2.FontFamily
+    3.FontAttributes
+    4.TextColor
+
+
+For Label Text Style,
+{% tabs %}
+{% highlight C# %}
```

Comment:
check code alignment

Reply:
**Change Label Text Style**
**Change Editor Text Style**

use 4# and not bold text